src/main/java/org/usfirst/frc4904/robot/commands/IndexerVelcroTurnSet.java

```diff
@@ -0,0 +1,25 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import edu.wpi.first.wpilibj.command.Command;
+
+public class IndexerVelcroTurnSet extends Command {
+	protected final boolean turned;
+
+	public IndexerVelcroTurnSet(boolean turned) {
+		super(turned ? "IndexerVelcroUp" : "IndexerVelcroDown");
+		requires(RobotMap.Component.Floorio.floorio.flipper);
```

Comment:
this can be onelined into super(turned ? "IndexerVelcroUp" : "IndexerVelcroDown", RobotMap.Component.Floorio.floorio.flipper)